src/common/feedGenerator.ts

```diff
@@ -192,7 +200,7 @@ export const selectRead = (
   const query = builder
     .select('1')
     .from(View, 'view')
-    .where(`view.userId = :userId`, { userId })
+    .where(`view.userId = :userId`, { userId: userId || '' })
```

Comment:
Since query is used in a lot of different ones as `EXISTS${subquery}` I added this to avoid querying with empty values.